ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -315,6 +315,20 @@ aaa_server_groups:
       - server: < host1_ip_address >
 ```
 
+#### CVX
+
+CVX cannot be enbled on a physical switch
+
+```yaml
+cvx:
+  shutdown: < true|false >
+  services:
+    mcs:
+      redis:
+        password: < password >
+        encryption: < true|false >
```

Comment:
To keep AVD consistent, we should always assume "encrypted" passwords.
Almost all other data models use `password: < encrypted_password >`. If it is important to support type 0, please use the same logic as under `tacacs_servers`, so something like:
```
password: < encrypted_password >
password_type: < 0 | 7 | 8a | default -> 7 >
```
```